apps/website/src/pages/create.js

```diff
@@ -14,6 +15,7 @@ export default function CreateRoute() {
       <Head>
         <title>Create</title>
       </Head>
+      <GlobalStyle />
```

Comment:
https://github.com/dev-launchers/dev-launchers-platform/pull/2110/files#r1826864273 sounds like we want to remove this because we migrated to tailwind. Why are we adding it back here?